versions/3.0.md

```diff
@@ -53,12 +53,12 @@ Additional utilities can also take advantage of the resulting files, such as tes
 		- [Callback Object](#callbackObject)
 		- [Headers Object](#headersObject)
 		- [Example Object](#exampleObject)
```

Comment:
If the `valueUrl` concept survives, consider whether having a name that is more similar to `externalDocs` which has some similarities here (non-$ref style references), like `externalValue` or `externalExample`